plugins/modules/fusion_hap.py

```diff
@@ -135,14 +135,11 @@
 def _check_iqn(module, fusion):
     hap_api_instance = purefusion.HostAccessPoliciesApi(fusion)
     hosts = hap_api_instance.list_host_access_policies().items
-    for host in range(0, len(hosts)):
-        if (
-            hosts[host].iqn == module.params["iqn"]
-            and hosts[host].name != module.params["name"]
-        ):
+    for host in hosts:
+        if host.iqn == module.params["iqn"] and host.name != module.params["name"]:
             module.fail_json(
                 msg="Supplied IQN {0} already used by host access polivy {1}".format(
```

Comment:
I know its not your code, but there is a typo here and your PR is in ideal position to fix it 😁

Reply:
Sure :)